test/src/multi_thread_matrix_calculation_test.cpp

```diff
@@ -482,5 +482,168 @@ TEST_F(TestMultiThreadCalculation, numSelfSubtractMatrix) {
     // make sure they are equal
     ASSERT_TRUE(equalSingleThread(singleOutput, multiOutput, 0, singleOutput.size()));
 }
+
+TEST_F(TestMultiThreadCalculation, matrixGreaterEqualMatrix) {
```

Comment:
follow the format of  `TEST_F(TestMultiThreadCalculation, matrixLessEqualMatrix)`
It can simplify your code and keep the formatting consistent